Doc/using/cmdline.rst

```diff
@@ -301,6 +305,18 @@ Miscellaneous options
       Modify ``.pyc`` filenames according to :pep:`488`.
 
 
+.. cmdoption:: -P
+
+   Don't add :data:`sys.path[0] <sys.path>`: don't prepend the current working
+   directory (``python -m module``), an empty string (``python -c code``) or
+   the first command line argument (``python script.py``) to :data:`sys.path`.
```

Comment:
```suggestion
   Do not prepend the initial working directory (``python -m module``), the
   current working directory (``python -c code``), or the directory of the
   executed script (``python script.py``) to :data:`sys.path`.
```
I think saying "prepend" suffices instead of talking about adding `sys.path[0]`. 

With the `-m` option, `sys.path[0]` is the full path of the initial working directory. With the `-c` option and the REPL, the current working directory is prepended to `sys.path`. It's an implementation detail that it's represented in this case as an empty string. It's not exactly an obscure implementation detail, since most people know that `normpath('')` -> `'.'`, but I think it's better to be explicit about the behavior.